.gitattributes

```diff
@@ -0,0 +1,6 @@
+# Prevent lock checksumming from getting tripped up by Windows
+# line-ending auto-conversion by git on checkout.
+/nox-support/*-reqs.txt -text
```

Comment:
This smells like the end result of a very annoying debugging process?

Reply:
No, just a guess / confidence that the only way for the lock to re-run only on Windows had to be line endings. I've never used `.gitattributes` before that I recall, but I do use dos2unix all the time / the Pex Windows project made me keenly aware of line endings.